### PR TITLE
Document new option behavior for hosting bundle

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/hosting-bundle.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ To obtain an earlier version of the installer:
 
 > [!NOTE]
 > For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).
+> When running the Hosting Bundle installer with options set, the value for each option will be saved in the registry. Subsequent installs from the same Major.Minor version band will use the same options, unless another set of options is explicitly passed from the command line. If the first install of the hosting bundle has no options passed, each option will get a default value of `0` written in to the registry.
```
Comment (jamshedd) [inline aspnetcore/host-and-deploy/iis/hosting-bundle.md:62]: This looks good. Could we also articulate what writing a default value of 0 implies (option command line option is treated as on or off)?

Comment (wtgodbe) [reply aspnetcore/host-and-deploy/iis/hosting-bundle.md]: Updated

**Review by jamshedd — APPROVED** [review]

File: aspnetcore/host-and-deploy/iis/hosting-bundle.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ To obtain an earlier version of the installer:
 
 > [!NOTE]
 > For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).
+> When running the Hosting Bundle installer with options set, the value for each option will be saved in the registry. Subsequent installs from the same Major.Minor version band will use the same options, unless another set of options is explicitly passed from the command line. If the first install of the hosting bundle has no options passed, each option will get a default value of `0` written in to the registry (a value of `0` implies that the option is off, meaning the user is not opting out of the given component).
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/hosting-bundle.md:62]:
```suggestion
> When running the Hosting Bundle installer with options set, the value for each option is saved in the registry. Subsequent installs from the same Major.Minor version band use the same options, unless another set of options is explicitly passed from the command line. If the first install of the hosting bundle has no options passed, each option gets a default value of `0` written in to the registry. A value of `0` implies that the option is off, meaning the user is not opting out of the given component.
```

**Review by Rick-Anderson — APPROVED** [review]
Per style guide, Avoid future case and parenthetical expressions.

File: aspnetcore/host-and-deploy/iis/hosting-bundle.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ To obtain an earlier version of the installer:
 
 > [!NOTE]
 > For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).
+> When running the Hosting Bundle installer with options set, the value for each option is saved in the registry. Subsequent installs from the same Major.Minor version band use the same options, unless another set of options is explicitly passed from the command line. If the first install of the hosting bundle has no options passed, each option gets a default value of `0` written in to the registry. A value of `0` implies that the option is off, meaning the user is not opting out of the given component.
```
Comment (halter73) [inline aspnetcore/host-and-deploy/iis/hosting-bundle.md:62]: Should this share the same `[!NOTE]` as the IIS Shared Configuration stuff?

Also, it seems like the fragment that `[ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration)` used to point to is now at `host-and-deploy/iis/advanced#aspnet-core-module-with-an-iis-shared-configuration` which is linked to from the `OPT_NO_SHARED_CONFIG_CHECK=1` bullet point, so I'm not sure that the above note still makes sense.

Comment (dougbu) [reply aspnetcore/host-and-deploy/iis/hosting-bundle.md:62]: Question on
> When running the Hosting Bundle installer with options set, &hellip;

Aren't settings (`0`s) saved the first time the installer runs _regardless_ of whether overrides are specified❔

Comment (wtgodbe) [reply aspnetcore/host-and-deploy/iis/hosting-bundle.md:64]: Moved the info to a separate note.

> Aren't settings (0s) saved the first time the installer runs regardless of whether overrides are specified❔

Yes, the first time, the options will always be saved. On subsequent installs, options are only saved if they are set.

Comment (dougbu) [reply aspnetcore/host-and-deploy/iis/hosting-bundle.md:64]: > Yes, the first time, the options will always be saved. On subsequent installs, options are only saved if they are set.

This isn't clear from "When running the Hosting Bundle installer with options set, &hellip;". But, it's not worth a follow-up PR.

**Pull request merged**